### PR TITLE
[Utility] Created clean_input to cleanly exit KeyboardInterrupts

### PR DESCRIPTION
**Pull request by Sma-Das**
### Background

Error messages aren't fun - so I created a simple function `clean_input` under `utils.py` that will cleanly exit in the event of a `KeyboardInterrupt`.
* Potentially add a callback argument for cleaner functions.

### Changes

* Created file: utils.py
* Created function under utils.py --> clean_input
* Replaced instances of `input` with `utils.clean_input`

### Test Plan

I feel its fairly standard error handling code. Even in that case, I provided some basic user testing: 
image

### Change Safety

- [ ] I have added tests to cover my changes
- [x] I have considered potential risks and mitigations for my changes

### CODE REVIEW

**Review by nponeccop — APPROVED** [review]

**Review by richbeales — APPROVED** [review]

File: scripts/utils.py
Comment (OannesGPT) [inline scripts/utils.py:1]: Would appreciate to get a message that I interrupted the script. ("You interrupted Auto-GPT")
```
 def clean_input(prompt: str=''):
    try:
        return input(prompt)
    except KeyboardInterrupt:
     print("You interrupted Auto-GPT")
        exit(0)

```

**Review by OannesGPT — DISMISSED** [review]

**Comment by nponeccop** [comment]
@Sma-Das Can you resolve the conflicts and add a message?

**Comment by Sma-Das** [comment]
All conflicts should be resolved

File: scripts/main.py [hunk 1]
```diff
@@ -133,12 +134,12 @@ def load_variables(config_file="config.yaml"):
...
     # Prompt the user for input if config file is missing or empty values
     if not ai_name:
-        ai_name = input("Name your AI: ")
+        ai_name = utils.clean_input("Name your AI: ")
         if ai_name == "":
             ai_name = "Entrepreneur-GPT"
 
-    if not ai_role:
-        ai_role = input(f"{ai_name} is: ")
+    if not ai_role:
```
Comment (nponeccop) [inline scripts/main.py]: Remove the trailing space (it's from conflicts)

**Review by nponeccop — APPROVED** [review]
Well done. print("You interrupted Auto-GPT") is an option

**Comment by Sma-Das** [comment]
Removed the whitespace and added the exit message

**Review by nponeccop — APPROVED** [review]

**Pull request merged**